engine/core/IgeEntity.js

```diff
@@ -88,6 +88,10 @@ var IgeEntity = IgeObject.extend({
 		this.streamSections(['transform']);
 	},
 
+	isRendering() {
```

Comment:
this format allows for sever to call and the `!!` will end up giving us `(false && !!undefined)` which will evaluate to `return true` when called from the server. Going to push a fix.

Reply:
it will not, if first part of the `&&` condition is false, the second never executes, this is basic js knowledge, you should know better

Reply:
yeah I was massively overthinking that.